### PR TITLE
feat(tooltip): default color to dark, change colors

### PR DESCRIPTION
**Pull request by yqrashawn**
---

## Checklist

- [x] Fix linting errors
- [x] Tests have been added / updated (or snapshots)

## Change information

1. fix style, padding, color
2. type change, SnippetTypes -> SnippetColors, DividerTypes -> DividerColors
3. rename prop in Snippet, Tooltip, Divider, Tag, type -> color

resolve CU-3tuy51

### CODE REVIEW

File: pages/en-us/components/divider.mdx [hunk 1]
```diff
@@ -132,19 +132,19 @@ A divider line separates different content.
...
-| **y**      | y-axis spacing    | `number` / `float`            | -                                | 2         |
-| **type**   | type of divier    | [DividerTypes](#dividertypes) | -                                | `default` |
-| **align**  | alignment of text | [DividerAlign](#divideralign) | -                                | `center`  |
-| **volume** | size of component | `number` / `float`            | -                                | 1         |
-| ...        | native props      | `HTMLAttributes`              | `'id', 'name', 'className', ...` | -         |
+| Attribute  | Description       | Type                            | Accepted values                  | Default   |
+| ---------- | ----------------- | ------------------------------- | -------------------------------- | --------- |
+| **x**      | x-axis spacing    | `number` / `float`              | -                                | 0         |
+| **y**      | y-axis spacing    | `number` / `float`              | -                                | 2         |
+| **color**  | type of divier    | [DividerColors](#dividercolors) | -                                | `default` |
```
Comment (AngeliaQ) [inline pages/en-us/components/divider.mdx]: description is better to be changed

File: pages/zh-cn/components/tooltip.mdx [hunk 2]
```diff
@@ -182,28 +176,30 @@ export const meta = {
...
-| ...                 | 原生属性                   | `HTMLAttributes`              | `'id', 'name', 'className', ...` | -         |
-
-<Attributes.Title>TooltipTypes</Attributes.Title>
+| 属性                | 描述                       | 类型                            | 推荐值                           | 默认      |
+| ------------------- | -------------------------- | ------------------------------- | -------------------------------- | --------- |
+| **text**            | 弹出框文字                 | `string` `React.ReactNode`      | -                                | -         |
+| **visible**         | 手动控制提示框的显示与隐藏 | `boolean`                       | -                                | `false`   |
+| **initialVisible**  | 初始是否可见               | `boolean`                       | -                                | `false`   |
+| **hideArrow**       | 隐藏箭头                   | `boolean`                       | -                                | `false`   |
+| **color**           | 不同的文字提示类型         | [TooltipColors](#tooltipcolors) | -                                | `default` |
```
Comment (AngeliaQ) [inline pages/zh-cn/components/tooltip.mdx]: 描述同步修改

**Comment by AngeliaQ** [comment]
Task linked: [CU-3tuy51 Tooltip component](https://app.clickup.com/t/3tuy51)

File: pages/zh-cn/components/tooltip.mdx (hunk 2 above)
Comment (yqrashawn) [reply pages/zh-cn/components/tooltip.mdx]: done

File: pages/en-us/components/divider.mdx (hunk 1 above)
Comment (yqrashawn) [reply pages/en-us/components/divider.mdx]: done

Comment (AngeliaQ) [reply pages/en-us/components/divider.mdx]: divier is error, maybe divider?

Comment (yqrashawn) [reply pages/en-us/components/divider.mdx]: done

**Review by AngeliaQ — APPROVED** [review]

**Review by jnoodle — APPROVED** [review]

**Pull request merged**